swagger.yaml

```diff
@@ -0,0 +1,249 @@
+swagger: "2.0"
+
```

Comment:
We usually don't put extra blank lines between objects. 
For example: https://github.com/osu-mist/web-api-skeleton/blob/master/swagger.yaml